paddle/fluid/framework/ir/mkldnn/multi_gru_seq_fuse_pass.cc

```diff
@@ -140,3 +141,7 @@ void MultiGruSeqFusePass::ApplyImpl(ir::Graph* graph) const {
 
 REGISTER_PASS(multi_gru_seq_fuse_pass,
               paddle::framework::ir::MultiGruSeqFusePass);
+REGISTER_PASS_CAPABILITY(multi_gru_seq_fuse_pass)
+    .AddCombination(
+        paddle::framework::compatible::OpVersionComparatorCombination().EQ(
+            "multi_gru", 0));
```

Comment:
Thank you for adding this check. Now please add `IsCompat()` method inside handler, to check declared compatibility and break fusion if requirements are not met.